lib/jira/install.js

```diff
@@ -1,28 +1,43 @@
 const { Installation } = require('../models')
 const getAxiosInstance = require('../jira/client/axios')
+const jwt = require('./util/jwt')
 
 module.exports = async (req, res) => {
   req.log('Received installation payload.')
   const { baseUrl: host, clientKey, sharedSecret } = req.body
-  res.sendStatus(204)
+  let successfulRes = false;
```

Comment:
Can we rename it to `requiresNewInstallation` or similar?

Reply:
Since we also need to update if its not the first installation, maybe I should change this to `requiresInstallionUpdate`?